### DISCUSSION TITLE
Trying to understand elevation values for forecasts

### DISCUSSION THREAD

**Discussion by carrbr** [weather-gov/api#769] [General]
Hi, I am trying to understand the elevation values returned by various APIs and webpages, which don't match for seemingly the same location. For example:

1. https://forecast.weather.gov/MapClick.php?lon=-121.44514560699466&lat=47.439350812150934, elevation = 2972 ft
2. https://forecast.weather.gov/MapClick.php?lat=47.4394&lon=-121.4451&unit=0&lg=english&FcstType=graphical, elevation = 4777 ft
3. https://api.weather.gov/points/47.4394,-121.4451, elevation = 4777 ft

I have noticed the same pattern for a number of other locations. If you click into a location via the map (1), the elevation reads much lower than if you click through to the hourly forecast (2) from that same page. If you access the forecast through the API (3), the elevation matches the hourly page (2). What's going on here? Are they operating on different grids? Is there some mistake in where the elevation data in 1 comes from? Something else? Thanks!

**Comment by dwhitemv25** [comment]
> Is there some mistake in where the elevation data in 1 comes from? 

Sorry, but I think you are misreading the page data. 2972 ft. is the elevation of the nearest observation station: 

> Current conditions at
> ALPENTAL EXIT (TALPE)
> Lat: 47.427°N Lon: 121.418°W Elev: 2972.0ft.

On Apple Maps, the lon/lat in the first URL (47.439350812150934,-121.44514560699466) appears near the summit of Denny Mountain, which has a listed elevation of 5545 ft. The Alpental exit from I-90 is a bit to the SE from there, but apparently in that mile or two of distance there is 2500 ft of elevation change. 

Looking at the topography, it looks like there is a severe amount of elevation change within the gridpoint, so whatever elevation is presented is likely an average.

**Comment by scadergit** [comment]
That is correct, and the grid elevation  on Mapclick is below the map.
